PureBasicIDE/AutoComplete.pb

```diff
@@ -1511,9 +1531,17 @@ Procedure AutoComplete_WordUpdate(IsInitial=#False)
         
       EndIf
       
-      
     EndIf
     
   EndIf
   
 EndProcedure
+
+; ****
+
+;- Added by mk-soft; Date 28.05.2020; Linux Redraw TreeView outside Scintilla Callback
```

Comment:
This comment is no longer necessary because Git stores this information about who added, changed, and removed which line.